cassandra-analytics-common/src/main/java/org/apache/cassandra/spark/data/partitioner/TokenPartitioner.java

```diff
@@ -51,7 +51,7 @@
 
 /**
  * Util class for partitioning Spark workers across the token ring
- * This class duplicates {@link org.apache.cassandra.spark.bulkwriter.TokenPartitioner} but is solely
+ * This class duplicates org.apache.cassandra.spark.bulkwriter.TokenPartitioner but is solely
```

Comment:
The change is unrelated. Let's revert and make it clickable.

Reply:
This class no longer depends on the core module and so the javadoc build fails.